index.js

```diff
@@ -1 +1,265 @@
+let students_body = document.getElementById('students_data_body');
+let students_button = document.getElementById('students');
+let courses_body = document.getElementById('courses_data_body');
+let courses_button = document.getElementById('courses');
+let studentsURL = 'https://code-the-dream-school.github.io/JSONStudentsApp.github.io/Students.json';
+let coursesURL = 'https://code-the-dream-school.github.io/JSONStudentsApp.github.io/Courses.json';
+let studentsInfo = [];
+let coursesInfo = [];
+let submitNewStudent = document.getElementById('submit_btn');
+let editStudentButton = document.getElementById('edit-student-submit');
+
+/*==========================================
+ Event Listeners
+===========================================*/
+students_button.addEventListener('click', () =>{
+  showStudents();
+})
+
+courses_button.addEventListener('click', () =>{
+  showCourses();
+})
+submitNewStudent.addEventListener('click', (e) => {
+  addNewStudent();
+})
+
+
+/*============================================
+ Getting all data and storing in one function
+==============================================*/
+async function allData() {
+  let response = await fetch(studentsURL);
+  studentsInfo = await response.json();
+  
+  studentsInfo.forEach(student => {
+    student.courses = [];
+  });
+  response = await fetch(coursesURL);
+  coursesInfo = await response.json();
+
+  coursesInfo.forEach(course => {
+    course.students = [];
+  });
+
+  document.getElementById('students').disabled = false;
+  document.getElementById('courses').disabled = false; 
+}
+
+/*======================================================
+STUDENTS BUTTON
+========================================================*/
+
+ //Checking students status-making display the green circle
+function checkStudentStatus(studentStatus){
+  if(studentStatus === true) {
+    return '<span title="active student" class="dot"></span>'
+  }else{
+    return '<span></span>'
+  }
+}
+
+//Displaying students cards with the data
+const showStudents = () =>{
+  document.getElementById('students_data_body').hidden = false;
+  document.getElementById('courses_data_body').hidden = true;
+
+  let options = ''
+  coursesInfo.forEach(course => {
+    options += '<option value="' + course.id + '">' + course.name +  '</option>';  
```

Comment:
Using simple concatenation is fine, however, for readability and ease in order to not keep track of every " or ', using template literals would be a better option.
  
options += `<option value="${course.id}">${course.name}</option>`;

This principle could be applied throughout the code.